modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -122,41 +121,41 @@ Your keystore contains 141 entries:
 (...)
 ----
 +
-To get the SHA1 hash of a certificate on the local filesystem:
+To get the SHA1 hash of a certificate on the local filesystem, run:
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
 $ openssl x509 -in __<certificate-file-path>__ -fingerprint -noout
 SHA1 Fingerprint=3F:DA:BF:E7:A7:A7:90:62:CA:CF:C7:55:0E:1D:7D:05:16:7D:45:60
 ----
 
-== Verification at the workspace level
+.Verifying imported certificates at the workspace level
 
-- Start a workspace, obtain the {orch-namespace} name in which it has been created, and wait for the workspace to be started.
+* Start a workspace, obtain the {orch-namespace} name in which it has been created and wait for the workspace to be started.
 
-- Get the name of the workspace Pod with the following command:
+* Get the name of the workspace Pod:
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
 $ {orch-cli} get pods -o=jsonpath='{.items[0].metadata.name}' -n __<workspace namespace>__ | grep '^workspace.*'
 ----
 
-- Get the name of the Che-Theia IDE container in the workspace Pod with the following command:
+* Get the name of the Che-Theia IDE container in the workspace Pod:
```

Comment:
Che-theia is just one of the IDEs that we support so we may want to change that, probably in another PR.